frontend/src/components/MapView/Layers/AnticipatoryActionLayer/index.tsx

```diff
@@ -33,15 +33,16 @@ import {
   useAAMarkerScalePercent,
   useMapCallback,
 } from 'utils/map-utils';
-import { getBoundaryLayerSingleton } from 'config/utils';
+import { getBoundaryLayersByAdminLevel } from 'config/utils';
 import {
   calculateAAMarkers,
   calculateCombinedAAMapData,
 } from 'context/anticipatoryActionStateSlice/utils';
 import { AAView } from 'context/anticipatoryActionStateSlice/types';
 import { Tooltip } from '@material-ui/core';
 
-const boundaryLayer = getBoundaryLayerSingleton();
+// Use admin level 2 boundary layer for Anticipatory Action
```

Comment:
In the future, we should make this configurable as the level might vary by deployment